src/cursor.c

```diff
@@ -1833,6 +1833,8 @@ void cursor_position(int x, int y, rfbClientPtr client) {
 		if (y >= dpy_y) y = dpy_y-1;
 	}
 
+	if (rotating)
+	    rotate_coords(x, y, &x, &y, screen->width, screen->height);
```

Comment:
this won't work for all rotations, I posted a comment about this in the issue - https://github.com/LibVNC/x11vnc/issues/152#issuecomment-807234776

Reply:
Alright., closing then.